livy/0.8.0-incubating-spark_3.0.1_2.12-hadoop_3.2.0_cloud/Dockerfile

```diff
@@ -6,6 +6,9 @@ LABEL maintainer="Aliaksandr Sasnouskikh <jaahstreetlove@gmail.com>"
 
 ENV BASE_IMAGE	$SPARK_BASE#$BASE_IMAGE
 
+RUN apt-get update --allow-releaseinfo-change && \
+    apt-get install curl -y
```

Comment:
Lets also add `jq`  , which is also missing in the image (and thus also brekas the documented example)